pages/cart.html

```diff
@@ -50,6 +49,7 @@ <h2 class="page-title">
 
   <main>
     <section id="cart-container" class="">
+      <h2 class="hide">hide</h2>
```

Comment:
The inner text should be more descriptive, like "Cart container"

Reply:
Toppen, tack för advice :))